.github/workflows/ci.yml

```diff
@@ -16,23 +16,23 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  poetry-install:
+  install:
```

Comment:
`install` sounds a bit close to a reserved word; perhaps worth making less generic just in case?

Reply:
Mhm it seems to work without any issues, what other name would you suggest?

Reply:
install-environment? install-python-environment?